packages/Button/stories/examples/Sizes.js

```diff
@@ -1,10 +1,9 @@
 import React from "react";
-import { ButtonStory } from "../../Button.stories.styles";
-import Button from "../../../src";
+import Button from "../../src";
 
 export default function CommonButtons() {
   return (
-    <ButtonStory>
+    <>
```

Comment:
minor: not fragment require

Reply:
Oh, I still need it, there are multiple `<div>` children.  But looking at it again, I don't like all those `<divs>`... 🤔